UPGRADE-1.8.md

```diff
@@ -1 +1,57 @@
 # UPGRADE FROM `v1.7.X` TO `v1.8.0`
+
+1. Add new bundles to your list of used bundles in `config/bundles.php` if you are not using it apart from Sylius:
+
+    ```diff
+    +   ApiPlatform\Core\Bridge\Symfony\Bundle\ApiPlatformBundle::class => ['all' => true],
+    +   Sylius\Bundle\ApiBundle\SyliusApiBundle::class => ['all' => true],
+    +   Lexik\Bundle\JWTAuthenticationBundle\LexikJWTAuthenticationBundle::class => ['all' => true],
+    ```
+
+2. Add configuration of ApiBundle in your `config/packages/_sylius.yaml` file:
+
+    ```diff
+        imports:
+    +       - { resource: "@SyliusApiBundle/Resources/config/app/config.yaml" }
+    ```
+
+3. Add configuration of new ApiBundle in your `config/packages/security.yaml` file:
+
+    ```diff
+        parameters:
+    +       sylius.security.new_api_admin_regex: "^/new-api"
+        
+        security:
+            providers:
+    +           sylius_admin_api_user_provider:
+    +               id: sylius.admin_user_provider.email_or_name_based
+            
+            firewalls:
+    +           new_api:
+    +               pattern: "%sylius.security.new_api_admin_regex%/.*"
+    +               stateless: true
+    +               anonymous: true
+    +               provider: sylius_admin_api_user_provider
+    +               json_login:
+    +                   check_path: /new-api/authentication-token
```

Comment:
Shouldn't this be also with regexp?
```
check_path: "%sylius.security.new_api_admin_regex%/authentication-token"

Reply:
It should. Would you like to open a PR?

Reply:
Done in: https://github.com/Sylius/Sylius/pull/11293